posts/2024-07-17_test4-live/README.md

```diff
@@ -0,0 +1,48 @@
+---
+publication_date: 2024-07-17T00:00:00Z
+slug: test4-live
+tags: [gnoland, ecosystem, govdao, test4]
+authors: [Kouteki]
+---
+
+# Blockchain Testnet Launched: Test4 Now Live
+
+We are excited to announce the launch of our blockchain testnet, Test4. This milestone marks a significant leap in our journey towards the gno.land mainnet, bringing new capabilities and stability to our development environment.
+
+## Key Features of Test4
+
+### Transition to a Decentralized Network
+
+Test4 is our first step towards a fully decentralized blockchain network. Unlike earlier stages, it supports multiple validator nodes, enhancing security and resilience. The initial validator set consists of:
+
+- Core engineering team
+- DevX team
+- Onbloc
+
+Which brings us to the second major feature, the GovDAO.
+
+### GovDAO
+
+The first iteration of the GovDAO enables on-chain voting and validator management. The initial GovDAO members are gno.land team members, major contributors and teams already building dapps on gno.land. The first voting proposals will be to add additional 3rd party validators:
```

Comment:
I would highlight this is a The first iteration of the testing GovDAO. In my opinion, we want to highlight this is not the final GovDAO structure and is solely for testing purposes at the moment

Reply:
Fixed in https://github.com/gnolang/blog/pull/103/commits/5a844c2a0d1a2fb116dcf8962e056163d1245738